todo_app/app.py

```diff
@@ -1,47 +1,40 @@
 from todo_app.data.session_items import delete_item
 from flask import Flask, render_template, request, url_for, redirect, session
 
+import requests
+
 from todo_app.flask_config import Config
 from todo_app.data import session_items as si
+from todo_app.classes import Item
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+# Index
 @app.route('/')
 def index():
     items = si.get_items()
     return render_template('index.html', items =items)
 
+# Add item 
 @app.route('/form', methods = ['GET','POST'])
```

Comment:
This shouldn't support GET.

Also the path "/form" is pretty generic - would be nicer for it to be something like "/create_item"